tests/test_order_LocalQl.py

```diff
@@ -4,6 +4,8 @@
 import unittest
 import util
 
+PERFECT_FCC_Q6 = 0.57452416
```

Comment:
Where does this value come from?

Reply:
@csadorf This is a test against a known value for an FCC system. We have verified this value by manually calculating it and using other packages like `pyboo`:

```python
from util import make_fcc
import boo
import freud
import numpy as np


box, positions = make_fcc(8, 8, 8)
nlist = freud.locality.AABBQuery(box, positions).query(positions, k=12, exclude_ii=True).toNList()

q6m = boo.bonds2qlm(positions, nlist[:], l=6)
q6 = boo.ql(q6m)
w6 = boo.wl(q6m)

# Using the median will ignore the surface values,
# since pyboo can't natively handle periodic systems
print(np.median(q6))  # 0.5745242597140698
print(np.median(w6))  # -0.0026260383340077588
```